configs/apcnet/apcnet.yml

```diff
@@ -0,0 +1,235 @@
+    lr schd: 80000
+  Results:
+    Task: Semantic Segmentation
+    Dataset: Cityscapes
+    Metrics:
+      mIoU: 79.64
+      mIoU(ms+flip): 80.61
+  Config: configs/apcnet/apcnet_r101-d8_512x1024_80k_cityscapes.py
+  Weights: https://download.openmmlab.com/mmsegmentation/v0.5/apcnet/apcnet_r101-d8_512x1024_80k_cityscapes/apcnet_r101-d8_512x1024_80k_cityscapes_20201214_115705-b1ff208a.pth
+- Name: apcnet_r50-d8_769x769_80k_cityscapes
+  In Collection: apcnet
+  Metadata:
+    method: APCNet
+    backbone: R-50-D8
+    crop size: 769x769
+    lr schd: 80000
+  Results:
+    Task: Semantic Segmentation
+    Dataset: Cityscapes
+    Metrics:
+      mIoU: 78.79
+      mIoU(ms+flip): 80.35
+  Config: configs/apcnet/apcnet_r50-d8_769x769_80k_cityscapes.py
+  Weights: https://download.openmmlab.com/mmsegmentation/v0.5/apcnet/apcnet_r50-d8_769x769_80k_cityscapes/apcnet_r50-d8_769x769_80k_cityscapes_20201214_115718-7ea9fa12.pth
+- Name: apcnet_r101-d8_769x769_80k_cityscapes
+  In Collection: apcnet
+  Metadata:
+    method: APCNet
+    backbone: R-101-D8
+    crop size: 769x769
+    lr schd: 80000
+  Results:
+    Task: Semantic Segmentation
+    Dataset: Cityscapes
+    Metrics:
+      mIoU: 78.45
+      mIoU(ms+flip): 79.91
+  Config: configs/apcnet/apcnet_r101-d8_769x769_80k_cityscapes.py
+  Weights: https://download.openmmlab.com/mmsegmentation/v0.5/apcnet/apcnet_r101-d8_769x769_80k_cityscapes/apcnet_r101-d8_769x769_80k_cityscapes_20201214_115716-a7fbc2ab.pth
+- Name: apcnet_r50-d8_512x512_80k_ade20k
+  In Collection: apcnet
+  Metadata:
+    method: APCNet
+    backbone: R-50-D8
+    crop size: 512x512
+    lr schd: 80000
+    inference time (ms/im):
+    - value: 50.99
+      hardware: V100
+      backend: PyTorch
+      batch size: 1
+      mode: FP32
+      resolution: (512x512)
+    memory (GB): 10.1
+  Results:
+    Task: Semantic Segmentation
+    Dataset: ADE20K
+    Metrics:
+      mIoU: 42.2
+      mIoU(ms+flip): 43.3
+  Config: configs/apcnet/apcnet_r50-d8_512x512_80k_ade20k.py
+  Weights: https://download.openmmlab.com/mmsegmentation/v0.5/apcnet/apcnet_r50-d8_512x512_80k_ade20k/apcnet_r50-d8_512x512_80k_ade20k_20201214_115705-a8626293.pth
+- Name: apcnet_r101-d8_512x512_80k_ade20k
+  In Collection: apcnet
+  Metadata:
+    method: APCNet
+    backbone: R-101-D8
+    crop size: 512x512
+    lr schd: 80000
+    inference time (ms/im):
+    - value: 76.34
+      hardware: V100
+      backend: PyTorch
+      batch size: 1
+      mode: FP32
+      resolution: (512x512)
+    memory (GB): 13.6
+  Results:
+    Task: Semantic Segmentation
+    Dataset: ADE20K
+    Metrics:
+      mIoU: 45.54
+      mIoU(ms+flip): 46.65
+  Config: configs/apcnet/apcnet_r101-d8_512x512_80k_ade20k.py
+  Weights: https://download.openmmlab.com/mmsegmentation/v0.5/apcnet/apcnet_r101-d8_512x512_80k_ade20k/apcnet_r101-d8_512x512_80k_ade20k_20201214_115704-c656c3fb.pth
+- Name: apcnet_r50-d8_512x512_160k_ade20k
+  In Collection: apcnet
+  Metadata:
+    method: APCNet
```

Comment:
whether the method filed is duplicate with In Collection

Reply:
Is it illegal?